pom.xml

```diff
@@ -1219,6 +1219,7 @@
     <activemqVersion>5.10.0</activemqVersion>
     <atomikosVersion>3.9.2</atomikosVersion>
     <camelVersion>2.14.1</camelVersion>
+    <elasticsearch-version>1.0.0_4</elasticsearch-version>
```

Comment:
> On Oct 6, 2015, at 2:32 PM, Seth Leger <notifications@github.com> wrote:
> 
> > +    <elasticsearch-version>1.0.0_4</elasticsearch-version>
I like the “-“ in XML for some reason as a personal preference but I gave up fighting that battle long about ;)

Thanks for review the code and working with Umberto, guys!